beacon/agent.py

```diff
@@ -147,3 +147,13 @@ def _set_project_root(self, project_root):
 
     def _set_source_version(self, source_version):
         self.source_version = source_version
+
+    def _get_grpc_channel(self):
+        """Returns a secure/insecure gRPC Channel instance"""
+        if self.dsn.has_secure_scheme():
+            ssl_credentials = grpc.ssl_channel_credentials()
```

Comment:
Where does it take the credentials from? Don't we need to seed the certificate somewhere in the package?

Reply:
Nope. For gRPC channel credentials, leave it empty or None. It will pick the certificate from the system. In later stages, will ship with a signed certificate from a CA.